src/sagas/repository.ts

```diff
@@ -851,7 +852,7 @@ export function* getColumnStats({ payload }: any): any {
         break;
       }
       case ColumnStatsRetrievalType.RETRIEVE_FILTERED_TEXT: {
-        const filteredResponse = yield call(authGet, filteredQuery);
+        const filteredResponse = yield call(authPost, baseQuery, { filter: filterStatement });
```

Comment:
Do we still need to check if the `filterStatement` is `undefined`?

Reply:
Oh yeah, good point. Thanks!